ql/src/test/queries/clientpositive/ctas_direct.q

```diff
@@ -0,0 +1,94 @@
+-- SORT_QUERY_RESULTS
+--! qt:dataset:alltypesorc
+
+set hive.support.concurrency=true;
+set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
+set hive.acid.direct.insert.enabled=true;
+set hive.exec.max.dynamic.partitions=200;
+set hive.exec.max.dynamic.partitions.pernode=200;
+
```

Comment:
Can you enable MetastoreDefault Transformer on this test? MetastoreDefaultTransformer is required to get the table location.

Reply:
I have added all MetadataTransformer tests in ctas_direct_with_transformers.q which contain CTAS queries with and without location specified wherein I have set MetadataTransformer as MetastoreDefaultTransformer. The other tests are without specifying the MetadataTransformer.

https://github.com/apache/hive/pull/3281/files#diff-3488c120c636fa4652899a6c639fd7818f2ff9e3abe354e48401d3fc6b73bc74